### PR TITLE
I love the smell of Rubocop fixes in the morning.

### PR DESCRIPTION
**Pull request by jaredmoody**
Low hanging fruit - style autocorrect fixes for #945

### CODE REVIEW

File: shoes-core/lib/shoes/oval.rb [hunk 1]
```diff
@@ -10,9 +10,9 @@ class Oval
     STYLES = { fill: Shoes::COLORS[:black] }
 
     def create_dimensions(left, top, width, height)
-      left   ||= @style[:left]
-      top    ||= @style[:top]
-      width  ||= @style[:diameter] || @style[:width] || (@style[:radius] || 0) * 2
+      left ||= @style[:left]
+      top ||= @style[:top]
+      width ||= @style[:diameter] || @style[:width] || (@style[:radius] || 0) * 2
```
Comment (PragTob) [inline shoes-core/lib/shoes/oval.rb]: hm, I do disagree with this - might be my personal style (and that my IDE makes those for me) but I like lining op `=` operators just as I like lining up `=>` in hashes, tremendously improves readability imo :)

File: shoes-core/spec/shoes/animation_spec.rb [hunk 2]
```diff
@@ -60,7 +60,7 @@
   it { is_expected.not_to be_stopped }
 
   describe "with framerate" do
-    let(:opts) { {:framerate => 36, :app => app} }
+    let(:opts) { { :framerate => 36, :app => app } }
```
Comment (PragTob) [inline shoes-core/spec/shoes/animation_spec.rb]: not sure about the spacious usage of whitespace in hash brackets either - I remember we talked about this, does someone know what we decided? @jasonrclark

File: shoes-package/spec/spec_helper.rb [hunk 3]
```diff
@@ -12,7 +12,7 @@
 require 'pathname'
 require 'rspec/its'
 
-Dir["#{SHOES_PACKAGE_SPEC_ROOT}/support/**/*.rb"].each {|f| require f}
+Dir["#{SHOES_PACKAGE_SPEC_ROOT}/support/**/*.rb"].each { |f| require f }
```
Comment (PragTob) [inline shoes-package/spec/spec_helper.rb]: also not too certain about the spaces here... but maybe it is indeed better and I should adapt :)

**Comment by PragTob** [comment]
Most of this looks good :+1: 

Personally I like aligning operators such as `=` for multiple assignmetns or something like that, much more readable.

The other things are about spaces around braces, where I'm not entirely sure.

Tobi

File: shoes-core/lib/shoes/oval.rb (hunk 1 above)
Comment (jaredmoody) [inline shoes-core/lib/shoes/oval.rb]: Thanks for pointing this out, I missed this and I agree with you. Looks like there's a configuration option called `MultiSpaceAllowedForOperators` that will allow lining these up. I'll drop this commit from the PR and make a new one for this cop.

File: shoes-package/spec/spec_helper.rb (hunk 3 above)
Comment (jaredmoody) [inline shoes-package/spec/spec_helper.rb]: I also prefer no space before my block parameters. Looks like there's a config option for this too, `SpaceBeforeBlockParameters`. I'll drop this commit and do a new PR for this too.

**Comment by jaredmoody** [comment]
Added some notes above, I'll move the questionable commits into their own PRs

File: shoes-core/lib/shoes/oval.rb (hunk 1 above)
Comment (PragTob) [inline shoes-core/lib/shoes/oval.rb]: :+1:  sounds good

**Comment by jaredmoody** [comment]
Rebased against master and bumped some method length/complexity limits from new code in master :P

**Comment by jaredmoody** [comment]
Rebased against master after `pre4` merge.

File: shoes-core/spec/shoes/animation_spec.rb (hunk 2 above)
Comment (jasonrclark) [inline shoes-core/spec/shoes/animation_spec.rb]: Don't recall a specific decision, but I can go either way on it so long as it's consistent between begin and end, which this does.

File: samples/expert-irb-adjusted.rb [hunk 4]
```diff
@@ -79,16 +79,16 @@ def run(str)
     when "\n"
       begin
         out, obj = IRBalike.run(@cmd + ';')
-        @str += ["#@cmd\n",
+        @str += ["#{@cmd}\n",
```
Comment (jasonrclark) [inline samples/expert-irb-adjusted.rb]: Didn't even realize this was allowed with strings! Glad to see it going away :grin:

**Comment by jasonrclark** [comment]
Reading through this, I'm happy with the hash changes as they stand so I'll take it.

**Pull request merged**